operator/pkg/controllers/configconnector/configconnector_controller.go

```diff
@@ -720,6 +723,9 @@ func (r *ConfigConnectorReconciler) applyValidatingWebhookConfigurationCustomiza
 
 // applyMutatingWebhookConfigurationCustomizationCR applies customizations specified in a MutatingWebhookConfigurationCustomization CR.
 func (r *ConfigConnectorReconciler) applyMutatingWebhookConfigurationCustomizationCR(ctx context.Context, cr *customizev1alpha1.MutatingWebhookConfigurationCustomization) error {
+	if err := checkForDuplicateWebhooks(cr.Spec.Webhooks); err != nil {
```

Comment:
IIUC, a bad webhook will "stop the world" e.g. we won't upgrade KCC.  I think that could be a valid decision, but it feels like a big blast radius.  Is my understanding correct?

Reply:
No it will not "stop the world". A bad webhook customization CR means it won't be applied by the operator, and the operator reports an error in the CR status.